packages/utils/src/utils.ts

```diff
@@ -1,3 +1,20 @@
+import { baseDecode } from './format';
+
 export function sortBigIntAsc(a: bigint, b: bigint) {
     return (a < b ? -1 : a > b ? 1 : 0)
-} 
\ No newline at end of file
+}
+
+
+export function keyToImplicitAddress(publicKey: string | { toString(): string }): string {
```

Comment:
`{ toString() : string }` should be replaced by `PublicKey` to be a bit more explicit on what the function is expecting